docs/UserGuide.adoc

```diff
@@ -377,57 +412,83 @@ TODO: ADD DETAILS ONCE IMPLEMENTED
 [[FilterStudent]]
 ==== Filter `filter`
 
-TODO: ADD DETAILS ONCE IMPLEMENTED
+Filter Student returns you all the students according to the given keywords below.
+
+You can filter students in a class using these two ways:
```

Comment:
then you can even remove this sentence.

Or better yet I think you should write something like this.

Filter student helps you find the students in a particular group and module. 

You can filter the students in Student View in the following ways: